packages/aws-cdk/README.md

```diff
@@ -549,6 +549,11 @@ $ cdk bootstrap --show-template > bootstrap-template.yaml
 $ cdk bootstrap --template bootstrap-template.yaml
 ```
 
+If have the need to encrypt the bootstrap artifacts, you can do this via the parameters `--bootstrap-ecr-key`,
+`--bootstrap-kms-key-id` and `--bootstrap-customer-key`. You have the possibility to provide own key id's or 
+let cdk cdk create one for you.
```

Comment:
I'd like to see this wording refined a bit more. There's some grammar issues in here.